packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -487,7 +487,16 @@ export abstract class FluidDataStoreContext
 		local: boolean,
 		localOpMetadata: unknown,
 	): void {
-		this.verifyNotClosed("process", true, extractSafePropertiesFromMessage(messageArg));
+		const safeTelemetryProps = extractSafePropertiesFromMessage(messageArg);
+		// On op process, tombstone error is logged in garbage collector. So, set "checkTombstone" to false when calling
+		// "verifyNotClosed" which logs tombstone errors. Throw error if tombstoned and throwing on load is configured.
```

Comment:
Shouldn't we just move the Tombstone Changed log to GC as well?

Reply:
Yeah I did but only for `process`. Other changed events are still in data store context. We should maybe move it to GC but they are not critical and it will require lots of changes. I wanted to keep this PR relatively small.

Reply:
Ok. I'm just nervous that we could make a mistake with the logic so spread out / diverse

Reply:
I agree but that's already the case. We can slowly converge it but doing it in one PR is too much.